pkg/kubelet/container/container_hash_test.go

```diff
@@ -68,7 +68,7 @@ var (
 }
 `
 
-	sampleV115HashValue = uint64(0x311670a)
+	sampleV115HashValue = uint64(0x9c2e6b2f)
 	sampleV116HashValue = sampleV115HashValue
```

Comment:
I'd suggest renaming this to `sampleV131HashValue` and dropping `sampleV116HashValue`